### PR TITLE
Employee firing/unfiring

### PR DESCRIPTION
**Pull request by TheBestTvarynka**

### CODE REVIEW

File: backend/core/src/main/java/com/feed_grabber/core/user/UserService.java [hunk 1]
```diff
@@ -220,7 +221,18 @@ public UUID createInCompany(UserRegisterInvitationDTO registerDto) throws Invita
 
     public void removeCompany(UUID id) {
         var userToUpdate = userRepository.getOne(id);
-        userToUpdate.setCompany(null);
+        userToUpdate.setIsDeleted(true);
+//        userToUpdate.setUsername(userToUpdate.getUsername() + " (fired)");
```
Comment (MaksymSlobodianyk) [inline backend/core/src/main/java/com/feed_grabber/core/user/UserService.java]: Delete commented code, please

File: frontend/src/sagas/users/sagas.ts [hunk 2]
```diff
@@ -54,10 +57,43 @@ function* changeUserRole(action) {
   }
 }
 
+function* loadFiredUserList(action: any) {
+  try {
+    // const query = action.payload;
+    const store = yield select();
+    const {page, size} = store.users.paginationFired;
+    // const api = query
+    //   ? `/api/user/search/?page=${page}&size=${size}&query=${query}`
```
Comment (MaksymSlobodianyk) [inline frontend/src/sagas/users/sagas.ts]: Delete commented code, please

File: backend/core/src/main/java/com/feed_grabber/core/user/UserService.java [hunk 3]
```diff
@@ -307,8 +319,8 @@ private int getDividerIndex(String usernameAndCompanyId) {
 //                .collect(Collectors.toList());
 //    }
```
Comment (MaksymSlobodianyk) [inline backend/core/src/main/java/com/feed_grabber/core/user/UserService.java:320]: Delete commented code, please

File: backend/core/src/main/resources/db/migration/V2020.09.07_14.30__alter_users_table.sql [hunk 4]
```diff
@@ -0,0 +1,4 @@
+-- alter table users drop constraint users_email_key;
+-- alter table users drop constraint users_username_key;
+
```
Comment (MaksymSlobodianyk) [inline backend/core/src/main/resources/db/migration/V2020.09.07_14.30__alter_users_table.sql]: Delete commented code, please

**Review by MaksymSlobodianyk — APPROVED** [review]
Delete all commented code and merge👍

File: backend/core/src/main/java/com/feed_grabber/core/user/UserService.java (hunk 3 above)
Comment (TheBestTvarynka) [reply backend/core/src/main/java/com/feed_grabber/core/user/UserService.java:320]: That's not my comment. So I wouldn't like to touch it)

**Pull request merged**